### ISSUE TITLE
the batch_norm layer problem

### ISSUE THREAD

**Issue by NanYoMy** [YipengHu/label-reg#22]
https://github.com/YipengHu/label-reg/blob/3fe086f8bb9d029acf9fdecba3cee92009cfb3dc/labelreg/layers.py#L40

Hi HU, 
I notices that the  **is_training** parameter of batch_norm layers(tf.contrib.layers.batch_norm) in your project  didn't set to **False** during the inference period. 
But according to the helper document of TensorFlow. this parameter should be set to False during the inference time.
https://www.tensorflow.org/api_docs/python/tf/contrib/layers/batch_norm

**Comment by YipengHu** [comment]
Thanks! You are absolutely right! However, in practice, it makes little
difference with one single pass on the test data, so it wasn't set only for
simplicity of the demo. This should be updated as you suggested.

**Comment by NanYoMy** [comment]
really appreciated!
